### PR TITLE
Shift key to add to current selection

### PR DESCRIPTION
**Pull request by irongollem**
Using the universal hotkey for "add more" I've implemented functionality that allows, when shift is held, the new selection to be added to the current instead of replacing this to support jagged selections or those across scrollable screens.

### CODE REVIEW

File: src/DragSelect.vue [hunk 1]
```diff
@@ -7,17 +7,36 @@
 </template>
 
 <script>
+  // Expands array to have a unique function that filters out non-unique values
+  // eslint-disable-next-line no-extend-native
+  Array.prototype.unique = function () {
```
Comment (stephan281094) [inline src/DragSelect.vue]: Instead of extending the Array prototype, let's create a function instead.

File: src/DragSelect.vue [hunk 2]
```diff
@@ -47,12 +66,13 @@
       },
       selectionBoxStyling () {
         // Only set styling when necessary
-        if (!this.mouseDown || !this.startPoint || !this.endPoint) return {}
+        if (!this.mouseDown || !this.startPoint || !this.endPoint) return {background: this.color}
```
Comment (stephan281094) [inline src/DragSelect.vue]: Please don't exceed 80 characters per line and make sure to use [standard.js](https://standardjs.com/) code style.

File: src/DragSelect.vue (hunk 1 above)
Comment (irongollem) [reply src/DragSelect.vue]: ok, will look into this.

File: src/DragSelect.vue (hunk 2 above)
Comment (irongollem) [reply src/DragSelect.vue]: Moved the return statement to its own line (bracketed) and added spacing around the object statement

File: src/DragSelect.vue (hunk 1 above)
Comment (irongollem) [reply src/DragSelect.vue]: Moved it to a method outside of the Vue object called uniqueArray and call this. Could/Should be moved to common section if this library would ever grow beyong the initial component

**Comment by irongollem** [comment]
@stephan281094 should be resolved. Please review again :) (beter zo toch?:P)

**Comment by irongollem** [comment]
@stephan281094 I actually implemented the changes requested almost a month ago. Please review

File: src/DragSelect.vue [hunk 3]
```diff
@@ -7,17 +7,35 @@
 </template>
 
 <script>
+  // Takes an aray and returns a copy of the array without duplicates
```
Comment (stephan281094) [inline src/DragSelect.vue]:
```suggestion
  // Takes an array and returns a copy of the array without duplicates
```

**Comment by stephan281094** [comment]
Thanks for the heads up and contributing! This is now featured in `0.2.0` 🎉

**Pull request merged**